internal/receiver/smartagentreceiver/output.go

```diff
@@ -60,6 +71,9 @@ func (output *Output) Copy() types.Output {
 
 func (output *Output) SendDatapoints(datapoints ...*datapoint.Datapoint) {
 	output.logger.Debug("SendDatapoints has been called.", zap.Any("datapoints", datapoints))
+	metrics, numDropped := output.converter.toMetrics(datapoints)
+	output.logger.Debug("SendDatapoints", zap.Any("metrics", metrics), zap.Int("numDropped", numDropped))
```

Comment:
We may want to remove the logger.Debug's in the final version so that Debug logs are not flooded for success cases. OK to keep for now if you are using them for debugging while you are actively developing.

Reply:
Good point, updated to use internal tracing and only debug log on dropped points.